src/hoi4_world/countries/hoi4_country_converter_tests.cpp

```diff
@@ -1575,4 +1575,97 @@ TEST(Hoi4worldCountriesCountryConverter, IdeologySupportDefaultsToAllNeutrality)
    ASSERT_TRUE(country_one.has_value());
    EXPECT_THAT(country_one->GetIdeologySupport(), testing::UnorderedElementsAre(testing::Pair("neutrality", 100)));
 }
+
+TEST(Hoi4worldCountriesCountryConverter, StartingResearchSlots)
```

Comment:
It's better to test behaviors than methods. Even though it's leads to a little repetition, this should be split into four different tests:

- DefaultCountriesGetThreeResearchSlots
- DecentralizedCountriesGetTwoResearchSlots
- GreatPowersGetFourResearchSlots
- UnrecognizedCountriesGetTwoResearchSlots